### ISSUE TITLE
Samples for Batch operations?

### ISSUE THREAD

**Issue by Dave76** [Azure/azure-cosmos-dotnet-v3#805]
Hey,

I have been playing with the new Batch class, but for some reason my response keeps coming back null.  Are there any samples available for Batch?  Here's my code:

```
        public async Task<BatchResponse> WriteBatchAsync<T>(string fileId, IEnumerable<T> items) where T: class, new()
        {
            // Arrange the batch
            var batch = _container.CreateBatch(new PartitionKey(fileId));

            var count = 0;
            foreach (var item in items) {
                batch = batch.CreateItem(item);
                count++;
            }

            _logger.WriteLine("Scaling up throughput");
            var throughputResponse = await _container.ReplaceThroughputAsync(MaxThroughput);
            _logger.WriteLine($"Scaled up throughput to {throughputResponse.Resource.Throughput}");

            var sw = Stopwatch.StartNew();
            var response = await batch.ExecuteAsync();
            sw.Stop();
            
            _logger.WriteLine("Scaling down throughput");
            var scaleDown = await _container.ReplaceThroughputAsync(MinThroughput);
            _logger.WriteLine($"Scaled down throughput to {scaleDown.Resource.Throughput}");

            return new BatchResponse
            {
                Count = count,
                ElapsedTime = sw.Elapsed
            };
        }
```

**Comment by Dave76** [comment]
Sorry, had a little issue with my debugger, I was getting a proper response, but still doesn't seem correct:

```
[
  {
    "StatusCode": 400,
    "IsSuccessStatusCode": false,
    "ETag": null,
    "ResourceStream": null,
    "RetryAfter": "00:00:00"
  },
  {
    "StatusCode": 424,
    "IsSuccessStatusCode": false,
    "ETag": null,
    "ResourceStream": null,
    "RetryAfter": "00:00:00"
  },
  {
    "StatusCode": 424,
    "IsSuccessStatusCode": false,
    "ETag": null,
    "ResourceStream": null,
    "RetryAfter": "00:00:00"
  },
  {
    "StatusCode": 424,
    "IsSuccessStatusCode": false,
    "ETag": null,
    "ResourceStream": null,
    "RetryAfter": "00:00:00"
  },
  {
    "StatusCode": 424,
    "IsSuccessStatusCode": false,
    "ETag": null,
    "ResourceStream": null,
    "RetryAfter": "00:00:00"
  }
]
```

**Comment by j82w** [comment]
@abhijitpai can you please take a look?

**Comment by ealsur** [comment]
Closed by #941
